src/main/java/teammates/ui/controller/InstructorCourseRemindAction.java

```diff
@@ -92,7 +99,14 @@ public ActionResult execute() throws EntityDoesNotExistException {
             }
 
             statusToUser.add(new StatusMessage(Const.StatusMessages.COURSE_REMINDERS_SENT, StatusMessageColor.SUCCESS));
-            redirectUrl = Const.ActionURIs.INSTRUCTOR_COURSE_DETAILS_PAGE;
```

Comment:
As said, changes in this `else` clause is not necessary because only the `INSTRUCTOR_COURSE_DETAILS_PAGE` has a remind all option.